dataclasses_json/cfg.py

```diff
@@ -41,30 +39,32 @@ def __init__(self):
 global_config = _GlobalConfig()
 
 
-def config(metadata: dict = None, *,
-           # TODO: these can be typed more precisely
-           # Specifically, a Callable[A, B], where `B` is bound as a JSON type
-           encoder: Callable = None,
-           decoder: Callable = None,
-           mm_field: MarshmallowField = None,
-           letter_case: Callable[[str], str] = None,
-           undefined: Optional[Union[str, Undefined]] = None,
-           field_name: str = None,
-           exclude: Optional[Callable[[str, T], bool]] = None,
-           ) -> Dict[str, dict]:
+def config(
+    metadata: dict = None,
+    *,
+    # TODO: these can be typed more precisely
+    # Specifically, a Callable[A, B], where `B` is bound as a JSON type
+    encoder: Callable = None,
+    decoder: Callable = None,
+    mm_field: MarshmallowField = None,
+    letter_case: Union[Callable[[str], str], 'LetterCase'] = None,
+    undefined: Optional[Union[str, Undefined]] = None,
+    field_name: str = None,
+    exclude: Optional[Callable[[str, T], bool]] = None,
+) -> Dict[str, dict]:
```

Comment:
Why do you change the existing lines indentation?

Reply:
I think it was because I formatted w/ black.